pkgs/applications/networking/dyndns/dyndnsc/default.nix

```diff
@@ -0,0 +1,64 @@
+{ stdenv, python3Packages }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "dyndnsc";
+  version = "0.5.1";
+
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    hash = "sha256-Sy6U0XhIQ9mPmznmWKqoyqE34vaE84fwlivouaF7Dd0=";
+  };
+
+  nativeBuildInputs = with python3Packages; [ pytestrunner ];
+  propagatedBuildInputs = with python3Packages; [
+    daemonocle
+    dnspython
+    netifaces
+    requests
+    setuptools
+  ];
+  checkInputs = with python3Packages; [ bottle pytest ];
```

Comment:
Please try to use pytestCheckHook.

Reply:
TIL! I should read the manual more often.